src/ui/public/angular-bootstrap/datepicker/year.html

```diff
@@ -1,15 +1,72 @@
-<table role="grid" aria-labelledby="{{uniqueId}}-title" aria-activedescendant="{{activeDateId}}">
+<table
```

Comment:
On the year selector I'd try to get your alignment correct (properly gridded). This is likely happening because your spacing for the day/date doesn't have the padding that you're putting on the "buttons". Because the year selector doesn't have the day name thead element it gets misaligned.

![image](https://cloud.githubusercontent.com/assets/324519/25353468/2d8964a8-28e4-11e7-9386-17ac9b29e3cb.png)